Formula/s/smpeg.rb

```diff
@@ -1,12 +1,14 @@
 class Smpeg < Formula
   desc "SDL MPEG Player Library"
   homepage "https://icculus.org/smpeg/"
-  url "svn://svn.icculus.org/smpeg/tags/release_0_4_5/", revision: "399"
+  url "https://github.com/icculus/smpeg/archive/refs/tags/release_0_4_5.tar.gz"
+  sha256 "e2e53bfd2e6401e2c29e5eb3929be0e8698bc9e4c9d731751f67e77b408f1f74"
+  license "LGPL-2.0-or-later"
   revision 1
 
   livecheck do
-    url "https://svn.icculus.org/smpeg/tags/"
-    regex(%r{href=.*?release[._-]v?([01](?:[._]\d+)+)/}i)
+    url :stable
+    regex(/^release[._-]v?(0(?:[._]\d+)+)$/i)
```

Comment:
```suggestion
    regex(/^release[._-]v?([01](?:[._]\d+)+)$/i)
    strategy :git do |tags, regex|
      tags.map { |tag| tag[regex, 1]&.tr("_", ".") }
    end
```

Unless I'm mistaken, `[01]` seems fine for the major version (based on how other SDL packages handle versions < 2).

This also adds a `strategy` block to replace the underscores in version text with dots.